packages/react-menu/src/common/isConformant.ts

```diff
@@ -7,6 +7,7 @@ export function isConformant<TProps = {}>(
   const defaultOptions: Partial<IsConformantOptions<TProps>> = {
     asPropHandlesRef: true,
     componentPath: module!.parent!.filename.replace('.test', ''),
+    skipAsPropTests: true,
```

Comment:
Can you please clarify why this is disabled?

Reply:
The `as` prop test should be reformulated. it's testing inappropriate  cases such as Function component, which is not supported anymore.